### ISSUE TITLE
Add SSO

### ISSUE THREAD

**Issue by dAnjou** [cbh123/shlinked#67]
Hi there,

I'd like to suggest adding SSO because it's not only more convenient but arguably more secure than rolling your own auth.

Bonus points for using LinkedIn as the first supported identity provider :D

Cheers

**Comment by cbh123** [comment]
Mm yeah this is a good point. It's something I've been thinking about but haven't gotten around to. Any thoughts to which provider?

**Comment by dAnjou** [comment]
Google or Facebook would be okay, or like I said LinkedIn itself :P

**Comment by cbh123** [comment]
True… although lower priority right now because the built in Phoenix
Auth (which I’m using) is secure and it would be a slight pain to add
SSO. Although if you think it would get more people to sign up, I could
prioritize higher !

On January 2, 2022, GitHub ***@***.***> wrote:
> Google or Facebook would be okay, or like I said LinkedIn itself :P
>

—
Reply to this email directly, view it on GitHub
<https://github.com/cbh123/shlinked/issues/67#issuecomment-1003738260>,
or unsubscribe <https://github.com/notifications/unsubscribe-
auth/ADL6M3XUBXQEBRQGBSCDWC3UUBZZ7ANCNFSM5DNHOIFQ>.
You are receiving this because you commented.Message ID:
***@***.***>
